core/src/main/java/org/apache/calcite/rex/RexCall.java

```diff
@@ -251,6 +251,10 @@ public List<RexNode> getOperands() {
     return operands;
   }
 
+  public int operandCount() {
```

Comment:
i might have been the one to suggest this, but is `getOperandCount` better since it follows typical getter naming convention

Reply:
That's right @tanclary, we considered that and I think Julian suggested we go with this method name to follow the `SqlCall` pattern:

`SqlCall` has `<S extends SqlNode> S operand(int)` and `int operandCount()`